projects/bbb_canbus_listener/main.cpp

```diff
@@ -203,9 +203,15 @@ int main(int argc, char **argv) {
 
                 std::map<std::string, NetworkTable::Value> values;
                 NetworkTable::Value gps_longitude;
-                double longitude = GET_GPS_LONG(frame.data);
+
+                // Perform a unit conversion from decimal degrees minutes to degrees.
+                double longitudeDDM = GET_GPS_LONG(frame.data);
+                int longitudeDD = longitudeDDM / 100;
+                double longitudeM = longitudeDDM - longitudeDD*100;
```

Comment:
smalllll thing haha, can you add a space between longitudeDD, *, and 100 and encapsulate in bracket so:
(longitude * 100) 
Just for clarity :))